zerberus/sfz.cpp

```diff
@@ -188,8 +193,15 @@ void ZInstrument::addRegion(SfzRegion& r)
                   }
             }
       Zone* z = new Zone;
-      r.setZone(z);
       z->sample = readSample(r.sample, 0);
+      if (z->sample) {
+            qDebug("Sample Loop - start %d, end %d, mode %d", z->sample->loopStart(), z->sample->loopEnd(), z->sample->loopMode());
+            if (r.loopStart == -1)
```

Comment:
Could you add a comment here? Something like read "loop from sample if no loop in defined in the region". Is this in the spec btw?